aslprep/data/boilerplate.bib

```diff
@@ -690,3 +690,111 @@ @article{wang2008empirical
   url={https://doi.org/10.1016/j.mri.2007.07.003},
   doi={10.1016/j.mri.2007.07.003}
 }
+
```

Comment:
It makes sense to add the 4S atlas citations here right? Instead of maintaining a separate bib file in AtlasPack?

Reply:
I think it's a good idea to have a bib file in the AtlasPack repo as well, just as something to check against, but if we want to describe the atlases in the boilerplate then we'll need the bibtex references in each prep repo anyway.